openpype/pipeline/workfile/workfile_template_builder.py

```diff
@@ -1426,6 +1473,174 @@ def cleanup_placeholder(self, placeholder, failed):
         pass
 
 
+class PlaceholderCreateMixin(object):
+    """Mixin prepared for creating placeholder plugins.
+
+    Implementation prepares options for placeholders with
+    'get_create_plugin_options'.
+
+    For placeholder population is implemented 'populate_create_placeholder'.
+
+    PlaceholderItem can have implemented methods:
+    - 'create_failed' - called when creating of an instance failed
+    - 'create_succeed' - called when creating of an instance succeeded
+    """
+
+    def get_create_plugin_options(self, options=None):
+        """Unified attribute definitions for create placeholder.
+
+        Common function for placeholder plugins used for creating of
+        publishable instances. Use it with 'get_placeholder_options'.
+
+        Args:
+            plugin (PlaceholderPlugin): Plugin used for creating of
+                publish instances.
+            options (Dict[str, Any]): Already available options which are used
+                as defaults for attributes.
+
+        Returns:
+            List[AbtractAttrDef]: Attribute definitions common for create
+                plugins.
+        """
+
+        creators_by_name = self.builder.get_creators_by_name()
+        print(creators_by_name)
```

Comment:
```suggestion
```

Reply:
Why this i resolved? Is there a reason to print creator classes?

Reply:
Resolved by mistake ;)